Header header

#Массив значений протечек на аппарате
bool[8] status